library/packages/src/modules/PackageCallbacks.rb

```diff
@@ -2989,9 +2989,6 @@ def layout_popup(message, button_box, info_on)
         HSpacing(50), # enforce width
         VSpacing(0.1),
         HBox(
```

Comment:
I think you can remove this `HBox` as it is not needed anymore. But double check how it would look...